.github/integration-in-memory-tests.yml

```diff
@@ -0,0 +1,18 @@
+# This file specifies the GitHub runner for each in-memory integration test and is utilized by .github/workflows/integration-in-memory-tests.yml CI workflow.
+#
+# Each entry in this file includes the following:
+# - The GitHub runner (runs_on field) that will execute tests.
+# - The tests that will be run by the runner.
+# - The triggers (e.g., PR Integration CCIP Tests) that should trigger these tests.
+#
+runner-test-matrix:
+  # START: CCIPv1.6 tests
+
+  - id: smoke/ccip/ccip_messaging_test.go:*
+    path: integration-tests/smoke/ccip/ccip_messaging_test.go
+    test_env_type: in-memory
+    runs_on: ubuntu-latest
+    triggers:
+      - PR Integration CCIP Tests
+    test_cmd: cd integration-tests/smoke/ccip && go test ccip_messaging_test.go -timeout 12m -test.parallel=2 -count=1 -json
```

Comment:
curious, why did you lower timeout and increase parallelization? Won't that, especially that latter change, impact stability?

Reply:
Lower timeout as the test expect to run faster complete way before that. Increase in parallel was not very intentional but I guess there won't be any impact as there is only one test in this runner for now.

Reply:
Why do you say this is an _increase_ in parallelism? The default value is already dynamic and calibrated to the number of cores, based on GOMAXPROCS:
> By default, -parallel is set to the value of GOMAXPROCS.

https://pkg.go.dev/cmd/go/internal/test